docs/v2-system-addon/data_dictionary.md

```diff
@@ -171,6 +171,7 @@ and losing focus. | :one:
 | `highlights_data_late_by_ms` | [Optional] Time in ms it took for Highlights to become initialized | :one:
 | `topsites_data_late_by_ms` | [Optional] Time in ms it took for TopSites to become initialized | :one:
 | `topsites_first_painted_ts` | [Optional][Service Counter][Server Alert for too many omissions] Timestamp of when the Top Sites element finished painting (possibly with only placeholder screenshots) | :one:
+| `topsites_icon_stats` | [Optional] Object with keys: "screenshot_with_icon", "screenshot", "tippytop", "rich_icon" and the value will be the number of TopSites with icons in the respective category | :one:
```

Comment:
nit: Each key in the `topsites_icon_stats` should be described separately here.

Also, please delete `topsites_size`, `topsites_screenshot`, and `topsites_tippytop`, as those are only for the test pilot.